source/js/RecipeCard.js

```diff
@@ -0,0 +1,251 @@
+class RecipeCard extends HTMLElement {
+  constructor() {
+    super();
+    let shadow = this.attachShadow({mode: 'open'});
+  }
+
+  set data(data) {
+    const style=`
```

Comment:
Codacy has a fix for the issue: [Operator '=' must be spaced.](https://app.codacy.com/gh/cse110-fa21-group27/cse110-fa21-group27/pullRequest?prid=8447291)
```suggestion
    const style = `
```